source/window/window.cpp

```diff
@@ -241,6 +241,8 @@ namespace hex {
         auto signalHandler = [](int signalNumber) {
             EventManager::post<EventAbnormalTermination>(signalNumber);
 
+            // Let's not loop on this...
+            std::signal(SIGABRT, nullptr);
             std::raise(SIGABRT);
```

Comment:
Why not re-raising the original signal here?

```suggestion
            std::signal(signalNumber, nullptr);
            std::raise(signalNumber);
```

Reply:
Yeah re-raising is probably smarter